kevlar/include/kevlar_bits/distribution/binomial.hpp

```diff
@@ -28,7 +28,7 @@ struct Binomial {
      * Computes the score of a binomial distribution with parameters n, p.
      * The score is given by:
      *      t - n * p
-     * where t is the sufficient statistic.
+     * where t is the count.
```

Comment:
This just seems clearer and more descriptive since we're talking specifically about a binomial.